docs/examples/DataSet/Accessing-data-in-DataSet.ipynb

```diff
@@ -0,0 +1,854 @@
+       " ParamSpecBase('y', 'numeric', 'Voltage', 'V'),\n",
+       " ParamSpecBase('y2', 'numeric', 'Current', 'A'))"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "interdeps.non_dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`what_depends_on` method allows to find what parameters depend on a given parameter:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Following parameters depend on 't' (Time, s):\n",
+      "- 'y' (Voltage, V)\n",
+      "- 'y2' (Current, A)\n"
+     ]
+    }
+   ],
+   "source": [
+    "t_ps = interdeps.paramspecs[2]\n",
+    "t_deps = interdeps.what_depends_on(t_ps)\n",
+    "\n",
+    "print(f'Following parameters depend on {t_ps.name!r} ({t_ps.label}, {t_ps.unit}):')\n",
+    "for t_dep in t_deps:\n",
+    "    print(f'- {t_dep.name!r} ({t_dep.label}, {t_dep.unit})')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Shortcuts to important parameters\n",
+    "\n",
+    "For the frequently needed groups of parameters, `DataSet` object itself provides convenient methods and properties.\n",
+    "\n",
+    "For example, use `dependent_parameters` property to get a only dependent parameters of a given `DataSet`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(ParamSpecBase('y', 'numeric', 'Voltage', 'V'),\n",
+       " ParamSpecBase('y2', 'numeric', 'Current', 'A'))"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dataset.dependent_parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is equivalent to:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(ParamSpecBase('y', 'numeric', 'Voltage', 'V'),\n",
+       " ParamSpecBase('y2', 'numeric', 'Current', 'A'))"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tuple(dataset.description.interdeps.dependencies.keys())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Note on inferences\n",
+    "\n",
+    "Inferences between parameters is a legacy feature that has not been used yet within QCoDeS. The initial concepts around `DataSet` included it in order to link parameters that are not directly dependent on each other as \"dependencies\" are. It is likely that \"inferences\" will be eventually deprecated and removed."
```

Comment:
> Inferences between parameters is a legacy feature that has not been used yet within QCoDeS.

This sentence is confusing to me.  A Legacy feature which has not been used yet?

Reply:
yes :) it's a feature that hasn't been used yet, and now we don't see where it will be used, so i already call it "legacy" until we deprecate it, and the remove it.